### PR TITLE
Improve `triggerKeyEvent()` warnings

### PR DESCRIPTION
**Pull request by Turbo87**
Resolves https://github.com/emberjs/ember-test-helpers/issues/368

/cc @mydea

### CODE REVIEW

File: addon-test-support/@ember/test-helpers/-utils.js [hunk 1]
```diff
@@ -35,3 +35,14 @@ export function runDestroyablesFor(object, property) {
...
+
+/**
+ Returns whether the passed in string consists only of numeric characters.
+
+ @private
+ @param {string} n input string
+ @returns {boolean} whether the input string consists only of numeric characters
+ */
+export function isNumeric(n) {
+  return !isNaN(parseFloat(n)) && isFinite(n);
```
Comment (cibernox) [inline addon-test-support/@ember/test-helpers/-utils.js]: Keycodes are only integers. I think you should use parseInt

File: addon-test-support/@ember/test-helpers/dom/trigger-key-event.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ const keyFromKeyCode = {
   18: 'Alt',
   20: 'CapsLock',
   27: 'Escape',
-  32: '',
+  32: 'Space',
```
Comment (cibernox) [inline addon-test-support/@ember/test-helpers/dom/trigger-key-event.js]: Why this change? I think I remember getting those values myself and being surprised by the empty string, but it was the value I saw when debugging

Comment (Turbo87) [reply addon-test-support/@ember/test-helpers/dom/trigger-key-event.js]: I got that value from https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode 🤔

File: addon-test-support/@ember/test-helpers/-utils.js (hunk 1 above)
Comment (Turbo87) [reply addon-test-support/@ember/test-helpers/-utils.js]: this seems to be the accepted solution for a lot of these questions on StackOverflow with lots of people mentioning that `parseInt()` should not be used for some reason.

ultimately this is only used to catch the case where someone used `'13'` instead of `13`. if someone would put in `'13.0'` in the current state it would also warn, but if we changed it to only allow integers it would no longer show that warning, so I think it is actually better this way

File: addon-test-support/@ember/test-helpers/dom/trigger-key-event.js (hunk 2 above)
Comment (cibernox) [reply addon-test-support/@ember/test-helpers/dom/trigger-key-event.js]: I created this which to test it and it seems to say otherwise: https://jsbin.com/qizayecoqe/edit?js,console,output

Comment (Turbo87) [reply addon-test-support/@ember/test-helpers/dom/trigger-key-event.js]: that is `' '`, not `''` though, so also something different. I'm fine with using `' '` instead.

Comment (cibernox) [reply addon-test-support/@ember/test-helpers/dom/trigger-key-event.js]: I thought I had a space, not an empty string. My bad.

Comment (Turbo87) [reply addon-test-support/@ember/test-helpers/dom/trigger-key-event.js]: updated to use `' '` now
